From: Chad Dupuis <cdupuis@redhat.com>
Date: Tue, 30 Nov 2010 20:01:32 -0500
Subject: [scsi] qla4xxx: Update version to V5.02.04.00.05.06-d0
Message-id: <20101130200132.10450.15207.sendpatchset@localhost.localdomain>
Patchwork-id: 29719
O-Subject: [RHEL 5.6 PATCH 28/28] qla4xxx: Update version number to
	V5.02.04.00.05.06-d0
Bugzilla: 656999

Bugzilla
--------

Bug 656999 (https://bugzilla.redhat.com/show_bug.cgi?id=656999)

Upstream Status
---------------

RHEL 5 specific.

Description
-----------

>From 185534eb65324cca09498481a7a6125cb26c22f4 Mon Sep 17 00:00:00 2001
From: Prasanna Mumbai <prasanna.mumbai@qlogic.com>
Date: Tue, 23 Nov 2010 14:21:05 -0800
Subject: [PATCH 29/29] V5.02.04.00.05.06-d0

Signed-off-by: Prasanna Mumbai <prasanna.mumbai@qlogic.com>

diff --git a/drivers/scsi/qla4xxx/ql4_version.h b/drivers/scsi/qla4xxx/ql4_version.h
index c1ee55b..7993abd 100644
--- a/drivers/scsi/qla4xxx/ql4_version.h
+++ b/drivers/scsi/qla4xxx/ql4_version.h
@@ -5,7 +5,7 @@
  * See LICENSE.qla4xxx for copyright and licensing details.
  */
 
-#define QLA4XXX_DRIVER_VERSION	"5.02.03.00.05.06-d1"
+#define QLA4XXX_DRIVER_VERSION	"5.02.04.00.05.06-d0"
 
 /*
  * Driver Versioning Scheme:
